api/server/router/volume/volume_routes.go

```diff
@@ -49,6 +51,13 @@ func (v *volumeRouter) postVolumesCreate(ctx context.Context, w http.ResponseWri
 	}
 
 	volume, err := v.backend.VolumeCreate(req.Name, req.Driver, req.DriverOpts, req.Labels)
+	if volumestore.IsAlreadyExists(err) {
+		version := httputils.VersionFromContext(ctx)
+		if versions.GreaterThan(version, "1.25") {
+			return httputils.WriteJSON(w, http.StatusNotModified, volume)
```

Comment:
Will here introduce a new API status code of 304?
If that, I am afraid we still need to update the API docs.